docs/backends.js

```diff
@@ -43,21 +43,26 @@ function backendOptions() {
       'circle': {
          name: 'Circle CI',
          url: 'https://circleci.com/api/v1/projects',
+         supportsOwnernameFiltering: false,
```

Comment:
I would make the `false` value implicit in all other backends to reduce clutter, and maybe shorten the option name to `ownernameFiltering`.

Reply:
Yeah, I thought about that, but in general I'm not a big fan of relying on implicit falseish values. But on the other hand, clutter is bad as well. I'll remove these and shorten the name.

Reply:
Generally it's good to have an explicit setting everywhere, but this whole feature is a quirk to work around a limitation (or even a bug) in Travis API. The other backends don't need to be saddled by this complication, which should go away once a migration to the current API is done, or a better workaround is found, or the problem fixed on the server side.

Reply:
True that.